examples/lighting-app/esp32/sdkconfig.defaults.esp32c2

```diff
@@ -0,0 +1,19 @@
+# Disable chip shell
+CONFIG_ENABLE_CHIP_SHELL=n
+
+# CONFIG_ESP32_WIFI_IRAM_OPT is not set
+# CONFIG_ESP32_WIFI_RX_IRAM_OPT is not set
+CONFIG_FREERTOS_PLACE_FUNCTIONS_INTO_FLASH=y
+CONFIG_FREERTOS_PLACE_SNAPSHOT_FUNS_INTO_FLASH=y
+
+CONFIG_COMPILER_OPTIMIZATION_ASSERTIONS_DISABLE=y
+
+CONFIG_LWIP_TCPIP_RECVMBOX_SIZE=16
+CONFIG_LWIP_TCP_RECVMBOX_SIZE=8
+
+CONFIG_BT_NIMBLE_ROLE_CENTRAL=n
+CONFIG_BT_NIMBLE_ROLE_OBSERVER=n
+
```

Comment:
`CONFIG_ESP_WIFI_SOFTAP_SUPPORT=n` ?

Reply:
target specific default configuratios, like `sdkconfig.defaults.esp32c2`, are applied on `sdkconfig.defaults`. So that file contains the `CONFIG_ESP_WIFI_SOFTAP_SUPPORT=n`, so no need to copy paste here.